src/docs/tutorials/set-up-a-tailscale-subnet-router.md

```diff
@@ -0,0 +1,212 @@
+---
+title: Set up a Tailscale Subnet Router
+---
+
+## What is a Subnet router?
+
+> A subnet router is a device within your tailnet that you use as a gateway that advertises routes for other devices that you want to connect to your tailnet without installing the Tailscale client.
+
+*Source: <a href="https://tailscale.com/kb/1019/subnets" target="_blank">Subnet routers</a> Via Tailscale's Documentation*
+
+In the context of Railway, The "other devices" are the services within a project.
+
+## About this Tutorial
+
+The goal of this tutorial is to enable us to connect to our production Postgres database without ever needing to expose it publicly.
+
+Since Railway doesn't currently offer a native way to access the <a href="https://docs.railway.app/reference/private-networking" target="_blank">private network</a> from our local environment, we can use Tailscale as a subnet router to accomplish this.
```

Comment:
Essentially moving this bad boy up so we don't burry the lede. (I do this all the time, dw.)

Reply:
Hmmm, I wouldn't know where to move it up to without changing the format of the guide, and I'd like to keep that the same as the other guides.